### DISCUSSION TITLE
[V5] 请问柱状图的y轴怎么设置最小刻度？

### DISCUSSION THREAD

**Discussion by PineSongCN** [antvis/G2#5768] [Q&A] [answered]
如图，单位是人，不应该出现小数
<img width="225" alt="image" src="https://github.com/antvis/G2/assets/39976617/68f205e4-b6cc-4159-b722-18d30b4ae001">

**Comment by pearmini** [comment, marked as answer]
可以过滤掉小数：

```js
chart.axis('y', { tickFilter: () => {} })
```
